cypress/e2e/page/annotation-pagetests.cy.js

```diff
@@ -58,10 +64,12 @@ describe("tests on annotation page ui with programmatic state loading and store
                     // 1. Load the app with test criteria using the dataset
                     cy.loadAppState("annotation", p_dataset, testCriteria);
 
+                    // cy.visit("/annotation");
```

Comment:
Why is this line still here? If it is meant as a reminder to re-enable once some other condition is met, then we should add a comment (e.g. TODO) that says this explicitly. That said, we have git for keeping track of old code, and keeping old code in comments for longer is not a good practice. A compromise we've done in the past is to make an issue to track the future re-implementation. That is probably the right solution here too.

Reply:
@jarmoza I think you added/commented this one, could you take a look, please?

Reply:
This is a leftover line from some debugging I was doing regarding store state. It can be removed.